airbyte-cdk/python/airbyte_cdk/sources/declarative/manifest_declarative_source.py

```diff
@@ -118,6 +129,9 @@ def read(
         self._configure_logger_level(logger)
         yield from super().read(logger, config, catalog, state)
 
+    def log_slice_message(self, logger: logging.Logger):
+        return self._emit_connector_builder_messages or super(self).log_slice_message(logger)
```

Comment:
log the slice message if `_emit_connector_builder_messages` or if the debug flag is turned on for consistency